lib/errors/error-collector.js

```diff
@@ -223,7 +229,15 @@ class ErrorCollector {
    * @param {Exception}  exception The Exception to be traced.
    */
   addUserError(transaction, error, customAttributes) {
-    if (!error) return
+    if (!error) {
```

Comment:
👍 Always braces on conditionals is the correct color for this perennial bikeshed.